site2/docs/io-debug.md

```diff
@@ -106,7 +106,7 @@ Use one of the following methods to get a connector log in localrun mode:
   
   ```
 
-To clearly explain the log information, here breaks down the large block of information into small blocks and add descriptions for each block.
+Following is a breakdown of the log into smaller blocks with descriptions to clearly explain the log information.
```

Comment:
The readability of the original one is better since it splits a long sentence into several parts.